src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
```

Comment:
Declare separate `enums` or union types
```suggestion
  shape: Shape;
  color: Color;
```